go.mod

```diff
@@ -6,4 +6,7 @@ require (
 	github.com/edsrzf/mmap-go v1.0.0
 	github.com/twitchyliquid64/golang-asm v0.0.0-20190126203739-365674df15fc
 	golang.org/x/sys v0.0.0-20190306220234-b354f8bf4d9e // indirect
+
 )
+
+replace golang.org/x/sys => github.com/golang/sys v0.0.0-20190306220234-b354f8bf4d9e
```

Comment:
could you explain why you need this `replace` directive?
it seems to point to the same commit than the one it replaces.

Reply:
For some reason, we can not visit and fetch any dependencies from `golang.org` in China.  A quick search, you can find many [projects](https://github.com/search?q=replace+golang.org%2Fx%2Fsys+%3D%3E+github.com%2F&type=Code) replaced that.

Reply:
ah, right.
could you add a comment about why this is added?
thanks.